mapf/src/motion/conflict.rs

```diff
@@ -148,7 +148,7 @@ where
         }
 
         // The final state should be almost exactly the same as the last move
-        assert!((to_state.point() - last_p).norm() < 1e-3);
+        assert!((to_state.point() - last_p).norm_squared() < 1e-6);
```

Comment:
I am curious why these assertions are necessary. Would such a case ever be reached?

Reply:
The point is precisely that the assertion should never fail, so if it ever panics here then there's a bug in the code and I'll find out before it causes confusing cascading problems beyond this point 😁

I'll be happy to remove the all the assertions in the library eventually, once the library has been heavily tested and before we do a major release. For now I think the added computational cost is negligible.